### DISCUSSION TITLE
messageCreate event not getting emitted

### DISCUSSION THREAD

**Discussion by marku782** [discordjs/discord.js#6443] [Q&A] [answered]
**Please describe the problem you are having in as much detail as possible:**
If I send a message after the bot gets ready, the messageCreate does not get emitted. 

**Include a reproducible code sample here, if possible:**

```js
const dc = require('discord.js');
const client = new dc.Client({intents: [dc.Intents.FLAGS.GUILDS]});
client.on('messageCreate', (a) => {
    console.log(a.content)
})
client.on('ready', () => {
    console.log('ready as ' + client.user.tag)
})
client.login(token)
```

**Further details:**

- discord.js version: 13.1.0 (stable)
- Node.js version: v16.6.2
- Operating system: GNU/Linux Pop!_OS, but can be reproduced on a different machine with Ubuntu 18.04.5 LTS
- Priority this issue should have – please be realistic and elaborate if possible: If this is not my fault, i would consider it quite high as messages are quite relevant for most bots

**Relevant client options:**

- partials: none
- gateway intents: GUILD

**Comment by vladfrangu** [comment, marked as answer]
You're missing the GUILD_MESSAGES intent alongside the GUILDS one

**Comment by marku782** [comment]
Same issue with the GUILD_MESSAGES intent.

**Reply by vladfrangu** [reply]
https://github.com/discordjs/discord.js/discussions/6443#discussioncomment-1186388

**Reply by marku782** [reply]
Yeah, that worked, thank you very much and sorry for not trying that out before opening an issue.

**Reply by vladfrangu** [reply]
It happens, don't worry! For questions, you should ask here or on our [Discord server](https://discord.gg/djs)!

**Reply by TrishamBP** [reply]
Hi I added in the GuildMessages, I am able to see the message logged on console but cannot see the message content
`const dotenv = require("dotenv");
// Loading up the discord bot access token.
dotenv.config();
// Logging the bot in
const { Client, GatewayIntentBits } = require("discord.js");
const client = new Client({
  intents: [
    GatewayIntentBits.Guilds,
    GatewayIntentBits.DirectMessages,
    GatewayIntentBits.GuildMessages,
  ],
});
// Listening to event emiited form the discord server
// Listening to the ready event when the bot logs in
client.on("ready", (event) => {
  console.log(`${event.user.username} has logged in`);
});
// Listening to the message event when somebody types or deletes a text
client.on("messageCreate", (message) => {
  console.log(message.content);
});
// Activating the bot in the newly created discord server
client.login(process.env.DISCORD_JS_BOTTOKEN);`

**Reply by monbrey** [reply]
This is a very old discussion - you also require the MessageContent intent in v14
